frontend/src/types.ts

```diff
@@ -670,7 +670,9 @@ export interface FilterType {
     returning_entity?: Record<string, any>
     target_entity?: Record<string, any>
     path_type?: PathType
+    include_event_types?: PathType[]
     start_point?: string | number
+    end_point?: string | number
```

Comment:
When can this be a number? Looking at the backend code, start and end points are both strings